J/JuliaInterpreter/Compat.toml

```diff
@@ -1,20 +1,39 @@
-["0-0.7"]
-julia = "1"
+["0 - 0.7"]
+julia = "1.0"
+# assuming exact match was wanted
+# otherwise, use ~ or ^ specifier
 
-["0.2-0.7.13"]
-CodeTracking = "0.3.2-0.5"
+["0.2 - 0.7.13"]
+CodeTracking = "0.3.2 - 0.5"
+# note this allows
+# CodeTracking [0.3.2 - 0.6)
+# otherwise use
+# CodeTracking = "0.3.2 - 0.5.8"
```

Comment:
```suggestion
["0.2-0.7.13"]
CodeTracking = "0.3.2-0.5"
```

Reply:
Since 0.5 allows versions up to but not including 0.6, doesn't this allow Interp 0.7.13 to be used with CodeTracking 0.5.9?

Line 13 implies  CodeT 0.5.9 requires Interp 0.7.14